explainers/audio-explainer.md

```diff
@@ -1,45 +1,76 @@
 # Audiobooks Explainer 
 ## What is an audiobook and why are we doing this?
-This sounds like an easy question, a “book with audio”! Audiobooks, for the purposes of the scope of our work right now, are publications that are primarily audio-based. Audiobooks can contain non-audio content like supplements or navigational documents, but the content is in audio format. 
+This sounds like an easy question, a “book with audio”! Audiobooks, for the purposes of the scope of our work right now, are publications that are primarily audio-based. Audiobooks can contain non-audio content like supplements or navigational documents, but their main content is in audio format. 
 
 Audiobooks have grown massively in popularity in the last few years despite being a segment of the industry for far longer. The main reason for this shift is due to several technologies:
-* Mobile devices (phones and tablets)
-* Improved mobile networks (3G, 4G LTE)
+* Pervasiveness of Mobile devices (phones and tablets)
 * Improved mobile storage capacity
+* Improved mobile networks (3G, 4G, LTE)
 * Widespread use of wifi
 
-When previously audiobook users would need several CDs to listen to their audiobook, they can now stream, download, and listen to their chosen content anywhere and time they choose. Audiobooks, like ebooks, are now in their pockets on demand. 
+When previously users would need several CDs to listen to one audiobook, they can now stream or download and listen to their chosen content anywhere, any time. Audiobooks, like ebooks, are now in their pockets on demand. 
 
-The main difference between ebooks and audiobooks currently is their level of specification. EPUB has existed in one form or another for almost 20 years, but audiobooks still do not have a common specification. A mature ebooks industry has co-opted audiobooks to make them work within the same contexts, but the lack of a specification has made this process arduous for user agents and by extension, users. 
+The main difference between ebooks and audiobooks is currently their level of specification. The EPUB format has existed as an open standard in one form or another for almost 20 years, but audiobooks still do not benefit from a standard format. A mature publishing industry has co-opted audiobooks to make them work within the same contexts, but the lack of standardization has made this process arduous for audiobook studios, distributors, retail platforms, user agents and by extension, users. 
+
+## Key use cases and current issues
+
+### Business to business
```

Comment:
Because this explainer is aimed at the TAG and we explain this further down in terms of packaging, I think I'd shorten this section (will add my own changes later but this is a good start).